doc/data/messages/t/too-many-arguments/good.py

```diff
@@ -0,0 +1,2 @@
+def foo(*args):
+    pass
```

Comment:
```suggestion
@dataclasses.dataclass
class ThreeDChessPiece:
    x : int
    y : int
    z : int
    type: str

def three_d_chess_move(white: ThreeDChessPiece, black: ThreeDChessPiece, current_player):
    pass
```
Imo using ``*args`` is making it even harder to know what's going on (which is the reason why pylint would not raise anymore: it don' t know what the args are). The real solution is a refactor, finding what arguments are semantically close together.

Reply:
True.. But the `bad.py` example doesn't really work with `black` :(

Reply:
What do you think of the latest suggestion ?